pyspecProcScripts/phasing.py

```diff
@@ -245,7 +300,11 @@ def filter_ranges(B, A):
     else:
         thebasename = ""
     # {{{ sign flip and average input for hermitian
-    input_for_hermitian = select_pathway(d, signal_pathway).C
+    if signal_pathway is not None:
+        input_for_hermitian = select_pathway(d, signal_pathway).C
+    else:
+        logger.info(strm("You are telling me that you did not apply phase cycling, so I am not selecting a coherence pathway"))
+        input_for_hermitian = d.C
```

Comment:
handle both data that has and has not been phase cycled

Reply:
why?

Reply:
I don't understand this -- it seems to me like if signal pathway is None, then you pull from the properties

Reply:
so you would have a variable for your pathway, that would be either pulled from the arg, or else pulled from the properties

Reply:
added the pulling from properties and an assert statement if that property is not set